pom.xml

```diff
@@ -95,10 +95,15 @@
             <artifactId>jcommander</artifactId>
             <version>1.82</version>
         </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+            <version>2.0.7</version>
```

Comment:
Could you make the version a property so we don't repeat it?

Reply:
Hello @kwart, done. Thank you.